manifest.php

```diff
@@ -23,6 +23,7 @@
  */
 
 use oat\generis\scripts\install\SetupDefaultKvPersistence;
+use oat\generis\scripts\install\TaskQueue;
```

Comment:
Should we change copyright year at the very top of this file to 2020?

Reply:
off: How you generally keep the copyright of the files updated? Only when there is a file modification? If we do not have an automatic way to do it, maybe it should be included in the Michael's checklist.